Client/Frontend/Browser/TabManagerStore.swift

```diff
@@ -128,19 +141,47 @@ class TabManagerStore {
         }
 
         var tabToSelect: Tab?
+        var fxHomeTab: Tab?
+        var customHomeTab: Tab?
+        
         for savedTab in savedTabs {
             // Provide an empty request to prevent a new tab from loading the home screen
             var tab = tabManager.addTab(flushToDisk: false, zombie: true, isPrivate: savedTab.isPrivate)
             tab = savedTab.configureSavedTabUsing(tab, imageStore: imageStore)
             if savedTab.isSelected {
                 tabToSelect = tab
             }
+            
+            if fxHomeTab == nil {
+                tab.isFxHomeTab ? fxHomeTab = tab : nil
```

Comment:
nit: Use the ternary to return the value to be assigned, not assign inside the ternary
`fxHomeTab = tab.isFxHomeTab ? tab : nil`